main.rb

```diff
@@ -0,0 +1,27 @@
+# require_relative 'app'
+
+# def main
+#   app = App.new
+#   interface
+#   selection = gets.chomp
+#   case selection
+#   when '1' then app.hello
+#   else
+#     puts 'Invalid selection'
+#     interface
+#   end
+# end
```

Comment:
- [ ] Here you are calling the hello method from the `app.rb` file and this method is not available on the `app.rb` file, the app is crashing. Please make sure to take care of all cases on this switch statement from 1 to 7 and call the appropriate method from the `app.rb` file over the selected option. (hint: if users select option 1 from the interface, call the `create_person` method in the switch statement from the `app.rb` file and so on for all options.)